src/NuGet.Clients/NuGet.PackageManagement.UI/Options/SourceMappingViewModel.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class SourceMappingViewModel
+    {
+        public string ID { get; private set; }
+        public IReadOnlyList<PackageSourceContextInfo> Sources { get; private set; }
+
+        public string SourcesString
+        {
+            get
+            {
+                return string.Join(", ", Sources.Select(s => s.Name));
+            }
+        }
+
+        public override string ToString()
+        {
+            return ID + " " + SourcesString;
```

Comment:
I was unable to drill-in to the controls using Narrator or NVDA, but I think it's important to announce the Sources List when visiting a list item. Let me know any better suggestions. Our Package Sources page also announces both Source Name and URL in one take.

The way I'm implementing this is a string concatenation. Narrator and NVDA pause briefly between each comma in the sources list. 

![image](https://user-images.githubusercontent.com/49205731/194179444-d6d65415-43b4-4091-8040-c69fed8244f0.png)